src/warm-cache-for-festival.ts

```diff
@@ -133,8 +125,25 @@ async function warm(festival: string, years: number[]) {
                 process.exit(1);
             }
 
+            const allDates: Date[] = data.days.map(day => day.date);
+
             // load the days/lineups, per the text format
             for (const day of data.days) {
+                day.date = new Date(day.date);
+
+                // Save day metadata
+                redisClient.hmset(`festival:${festival.toLowerCase()}_${year}:days:${day.number}`,
+                                  {
+                                      number : day.number.toString(),
```

Comment:
This probably doesn't need to be stringified but I'm ambivalent since it's redundant already because it's in the key needed to even query for this object.